README.md

```diff
@@ -0,0 +1,93 @@
+# Atlas Basic Terraform Module
+
+This Terraform module creates a basic setup on [MongoDB Atlas] [atlas]. 
+
+It creates the following resources:
+
+- A MongoDB Atlas project.
+- A MongoDB Atlas cluster provisioned with AWS, GCP or AZURE provider. It can also be a free tier (TENANT) cluster.
+- Zero or more IP addresses.
+- Zero or more CIDR blocks.
+- One or more database users.
```

Comment:
I think the sequence is:
project, then users, then IP addresses, then CIDR blocks then cluster